pkg/minikube/constants/constants_kubeadm_images.go

```diff
@@ -21,19 +21,19 @@ var (
 		"v1.24": {
 			"coredns/coredns":         "v1.8.6",
 			"etcd":                    "3.5.1-0",
-			"kube-apiserver":          "v1.23.0",
-			"kube-controller-manager": "v1.23.0",
-			"kube-proxy":              "v1.23.0",
-			"kube-scheduler":          "v1.23.0",
+			"kube-apiserver":          "v1.23.1",
```

Comment:
I have been thinking since these seem to get updated more than we thought, maybe we should have a Per Kubernetes Patch Version ?

Reply:
wdyt @afbjorklund

Reply:
it looks broken ? it maps v1.24 to v1.23.1 ?

Reply:
maybe what is available at the time PR was created?